client/my-sites/plans-features-main/index.tsx

```diff
@@ -454,7 +453,7 @@ const PlansFeaturesMain = ( {
 	let _customerType = chooseDefaultCustomerType( {
 		currentCustomerType: customerType,
 		selectedPlan,
-		currentPlan: { productSlug: currentPlan?.productSlug },
+		currentPlan: { productSlug: currentPlan?.productSlug || PLAN_FREE },
```

Comment:
Why is this necessary? If `currentPlan` exists as `undefined`, then we should probably update the method to work with `undefined` values.

Reply:
Good catch! It wasn't necessary and I have removed it.